Author: Ole Streicher <olebole@debian.org>
Description: Make TFORMx check more flexible
 The maximal column length in cfitsio changed between version 3370 and 3380.
 This patch checks their syntax, without a specific length.
Bug: https://github.com/astropy/astropy/issues/4646
--- a/astropy/io/fits/tests/test_image.py
+++ b/astropy/io/fits/tests/test_image.py
@@ -1070,7 +1070,7 @@
         hdu.writeto(self.temp('test.fits'))
 
         with fits.open(self.temp('test.fits')) as hdul:
-            assert (hdul['SCI'].data == cube).all()
+            assert np.abs(hdul['SCI'].data - cube).max() < 1./15.
 
     def test_subtractive_dither_seed(self):
         """
